api/src/main/java/com/graphhopper/util/Helper.java

```diff
@@ -318,23 +318,24 @@ public static double keepIn(double value, double min, double max) {
     }
 
     /**
-     * Round the value to the specified exponent
+     * Round the value to the specified number of decimal places, i.e. decimalPlaces=2 means we round to two decimal
+     * places. Using negative values like decimalPlaces=-2 means we round to two places before the decimal point.
      */
-    public static double round(double value, int exponent) {
-        double factor = Math.pow(10, exponent);
+    public static double round(double value, int decimalPlaces) {
+        double factor = Math.pow(10, decimalPlaces);
         return Math.round(value * factor) / factor;
     }
 
     public static double round6(double value) {
-        return Math.round(value * 1e6) / 1e6;
+        return round(value, 6);
```

Comment:
Was there any reason these methods did not delegate to round() before?

Reply:
No, I think I initially thought two methods (round4 and round6) would be sufficient. But not 100% sure.